packages/at_chops/pubspec.yaml

```diff
@@ -4,7 +4,7 @@ version: 1.0.2
 repository: https://github.com/atsign-foundation/at_libraries
 
 environment:
-  sdk: '>=2.18.3 <3.0.0'
+  sdk: '>=2.15.1 <3.0.0'
```

Comment:
you should add changelog and increment at_chops version in pubspec

Reply:
Updated